app/picture-answer/picture-answer.css

```diff
@@ -0,0 +1,8 @@
+button {
```

Comment:
Always always always style your components with rules like 

```
picture-answer button {

}
```

Never, never like 

```
button {

}
```

The latter styles *every button in the app*, including ones in other components. The text-answer folks will be very mad at you if you mess up their styles.

Even better would be to add a class to that button, and do

```
picture-answer .answer-button {

}
```